src/components/_button.scss

```diff
@@ -49,13 +53,20 @@
     @include sui-button--full;
   }
 
-  @if $color != null {
-    @include sui-button--custom($color);
+  @if $bgcolor != null and $c-color != null {
+    @include sui-button--custom($bgcolor, $c-color);
```

Comment:
Wrong named param: `$c-color`, regarding the mixin definition it should be `$color`.